corehq/apps/reports/standard/deployments.py

```diff
@@ -477,12 +494,51 @@ def _fmt_timestamp(timestamp):
 
         for row in table[1:]:
             # Last submission
-            row[len(location_colums) + 1] = _fmt_timestamp(row[len(location_colums) + 1])
-            # Last sync
             row[len(location_colums) + 2] = _fmt_timestamp(row[len(location_colums) + 2])
+            # Last sync
+            row[len(location_colums) + 3] = _fmt_timestamp(row[len(location_colums) + 3])
         result[0][1] = table
         return result
 
+    def get_location_column(self, user_doc, user_loc_dict):
+        user = CouchUser.wrap_correctly(user_doc)
+        if not user.get_location_ids(self.domain):
+            return '---'
+        return self._get_formatted_assigned_location_names(user, user_loc_dict)
+
+    def _get_formatted_assigned_location_names(self, user, user_loc_dict):
+        """
+        Create an HTML formatted string of the given assigned location names.
+        The primary location will be highlighted in bold.
+        """
+        assigned_location_ids = user.get_location_ids(self.domain)
+        primary_location_id = user.get_location_id(self.domain)
+        formatted_loc_names = []
+        for loc_id in assigned_location_ids:
+            loc_name = user_loc_dict.get(loc_id)
+            if loc_id == primary_location_id:
+                formatted_loc_names.insert(
+                    0, f'<strong>{loc_name}</strong>'
+                )
+            else:
+                formatted_loc_names.append(loc_name)
+
+        formatted_str = ', '.join(formatted_loc_names[:4])
+        html_nodes = [
+            f'<span class="locations-list">{formatted_str}</span>',
+        ]
+        if len(formatted_loc_names) > 4:
+            all_str = ', '.join(formatted_loc_names)
+            view_controls_html_nodes = [
+                f'<span class="loc-view-control">{_("...See more")}</span>',
+                f'<span class="loc-view-control" style="display:none">{_("...See less")}</span>',
```

Comment:
nit: I am not sure I have seen "See less" on such widgets before. Is "Collapse" a better word?

Reply:
That's a good point. I have updated the text in [b045a3d](https://github.com/dimagi/commcare-hq/pull/34744/commits/b045a3dc300e3d96e2d863eaecd37f4d544da471)